LDMP/jobs/mvc.py

```diff
@@ -192,7 +202,37 @@ def updateEditorGeometry(
         editor.setGeometry(option.rect)
 
 
-class DatasetEditorWidget(QtWidgets.QWidget, WidgetDatasetItemUi):
+class DatasetActions:
```

Comment:
Please do not overuse inheritance (and even more severe, multiple inheritance). IMO this is a fault of the original plugin. It makes the code harder to read and there is no clear advantage in this case.

In this case we do not need this `DatasetActions` class at all, as its methods can go directly in the `DatasetDetailsDialog` class.

Just to clarify, I am not against using inheritance, where it makes sense. There is currently no other code that needs the methods offered by `DatasetActions` so there is no need for this kind of abstraction.